shapely/ops.py

```diff
@@ -310,24 +281,18 @@ def nearest_points(g1, g2):
 
     The points are returned in the same order as the input geometries.
     """
-    seq = lgeos.methods['nearest_points'](g1._geom, g2._geom)
+    seq = pygeos.shortest_line(g1, g2)
```

Comment:
I was about to ask about why `nearest_points` wasn't in PyGEOS, but then I see https://github.com/pygeos/pygeos/pull/334

Reply:
Yeah, we will need to decide on whether we want to expose both versions top-level in Shapely 2.0 (but that doesn't need to be done in this PR)